development/internationalization/translation/using-the-translator.md

```diff
@@ -106,6 +106,13 @@ If you have have replacements to peform in your wording, then there are two opti
     <div>{l s='There are %products_count% items in your cart.' sprintf=['%products_count%' => $cart.products_count] d='Shop.Theme.Checkout'}</div>
     ```
 
+**Note:** The `l` macro does not support escaping the strings for javascript. Instead you can assign the translation to a variable and escape it afterwards:
+
+```html
```

Comment:
Try `smarty` instead of `html`

Reply:
Sorry, didn't notice it's merged already. Afaik markdown codeblock supports smarty syntax and it's .tpl file, not .html so I simply suggest using correct syntax.